### PR TITLE
[romyrobot][WIP] Initial contribution

### PR DESCRIPTION
**Pull request by xeniter**
# ROMY robot binding

Binding for your romy robot vacuum cleaner

# Description

https://community.openhab.org/t/romy-robot-integration-via-http-binding-austrian-vacuum-robot-highly-recommended/143307

### CODE REVIEW

File: bundles/org.openhab.binding.romyrobot/pom.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>4.1.0-SNAPSHOT</version>
```
Comment (wborn) [inline bundles/org.openhab.binding.romyrobot/pom.xml:10]:
```suggestion
    <version>4.2.0-SNAPSHOT</version>
```

**Review by lsiepel — CHANGES_REQUESTED** [review]
As it is a WIP, i had a very quick scan and did not do an extensive review. 

Just as a reminder; also run the i18n pluging, write the readme contents and adjust all the loggers to the right level if even needed to log at all.

**Comment by lolodomo** [comment]
I added the tag "work in progress" as "WIP" is still in the PR title.
Let us know when this is no more "work in progress".

**Comment by lsiepel** [comment]
Gentle ping @xeniter we are about to reach the one year marker without any change to this PR. Really hope you are ok and are able to continue work on this. Please let me know if you want to proceed with this PR. Eventually it might be closed due to inactivty.

**Comment by lsiepel** [comment]
Closing due to inactivty. The code is not lost, just no longer directly visible from this repository. If someone decides to continue, please do. Me or another maintainer will be happy to review it as long as there is progress.

**Pull request closed without merging**